### PR TITLE
Feature/fonts

### PR DESCRIPTION
**Pull request by swift-student**
- Muli and Montserrat custom fonts added to project
- Static functions for creating these fonts programmatically added to Fonts.swift in helpers
- Unit tests for initializing these fonts created in FontTests

### CODE REVIEW

File: EcoSoapBank/Helpers/Fonts.swift [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        case regular = "-Regular"
+        case extraBold = "-ExtraBold"
+        case extraLightITalic = "-ExtraLightItalic"
+        case black = "-Black"
+        case semiBold = "-SemiBold"
+    }
+    
+    static func montserrat(ofSize size: CGFloat, style: MontserratStyle = .regular) -> UIFont {
+        UIFont(name: "Montserrat\(style.rawValue)", size: size)!
+    }
```
Comment (DeVitoC) [inline EcoSoapBank/Helpers/Fonts.swift:53]: I like these Enums and methods for making it simpler to select a custom font.

**Comment by swift-student** [comment]
I also should have said that we likely won't need all these styles, but I didn't want to pare it down before we knew exactly what styles we would use in the design.

File: EcoSoapBankTests/FontTests.swift [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+//  EcoSoapBankTests
+//
+//  Created by Shawn Gee on 8/10/20.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import XCTest
+@testable import EcoSoapBank
+
+class FontTests: XCTestCase {
```
Comment (jonbash) [inline EcoSoapBankTests/FontTests.swift:12]: Yay tests; I need to write those...

File: EcoSoapBank/Helpers/Fonts.swift (hunk 1 above)
Comment (jonbash) [reply EcoSoapBank/Helpers/Fonts.swift:53]: Agreed.

**Pull request merged**